### PR TITLE
New Kusama Guide Homepage

### PR DESCRIPTION
**Pull request by krichard410**
WIP - Creating a new Kusama Guide Homepage
Still need to: 
- [x] Clean up code (custom.css, index.js)
- [x] Fix CSS (font, footer styles, nav styles, button width)
- [x] Add a different header image 
- [x] Figure out why extra files were edited and added to this PR
- [x] Translate text functionality

To run, 
`yarn install`
`yarn kusama:start`

### CODE REVIEW

**Comment by lsaether** [comment]
It looks like the old files from the branch before it was merged as PR are included here, can you remove them so it's only the files you changed that are included?

File: docs/maintain-guides-how-to-join-council.md [hunk 1]
```diff
@@ -57,6 +57,5 @@ left column under the heading "Members."
 ![e](assets/council/member.png)
 
 Congratulations! Now you are able to participate on the council by making motions or vetoing
-proposals. It's a good idea to now [set a nick](https://guide.kusama.network/en/latest/try/nicks/)
```
Comment (krichard410) [inline docs/maintain-guides-how-to-join-council.md:60]: These two docs files `docs/maintain-guides-how-to-join-council.md` and `docs/maintain-guides-secure-validator.md` aren't actually changing from the already published ones, this PR hadn't pulled it from the master.

File: kusama-guide/pages/en/index.js [hunk 2]
```diff
@@ -1,19 +1,110 @@
...
-          window.location.href = 'docs/en/kusama-index';
-        `,
-        }}
-      />
-      If you are not redirected automatically, follow this <a href="docs/en/kusama-index/">link</a>.
-    </div>
-  );
+class HomeSplash extends React.Component {
+  render() {
+    // const {siteConfig, language = ''} = this.props;
```
Comment (lsaether) [inline kusama-guide/pages/en/index.js]:
```suggestion
    // const {siteConfig, language = ''} = this.props;
```

Remove if not needed

File: kusama-guide/siteConfig.js [hunk 3]
```diff
@@ -4,23 +4,23 @@ const isBuilding = process.env.BUILDING;
...
   tagline: 'One-stop-shop for Kusama information.',
-  url: isPub ? "https://guide.kusama.network" : "", // Your website URL
-  baseUrl: isBuilding ? baseUrlPattern : "/",
-  projectName: isPub ? "kusama-guide-hosting" : "",
-  organizationName: 'w3f',
-  cname: "guide.kusama.network",
+  url: isPub ? "https://lsaether.github.io/" : "", // Your website URL
+  baseUrl: isBuilding ? baseUrlPattern : (isPub ? "/test/" : "/"),
+  projectName: isPub ? "test" : "",
+  organizationName: 'lsaether',
```
Comment (lsaether) [inline kusama-guide/siteConfig.js]:
```suggestion
   url: isPub ? "https://guide.kusama.network" : "", // Your website URL
  baseUrl: isBuilding ? baseUrlPattern : "/",
  projectName: isPub ? "kusama-guide-hosting" : "",
  organizationName: 'w3f',
  cname: "guide.kusama.network",
```

This was my testing configuration, replaced now with the live settings.

**Review by lsaether — COMMENTED** [review]
Was the bird supposed to be sightly off-kilter on the splash? It shows slightly to the left on my screen.

image

**Review by lsaether — APPROVED** [review]
LGTM

**Pull request merged**